### DISCUSSION TITLE
1000 genomes and UK biobank already in tskit format

### DISCUSSION THREAD

**Discussion by giulioisac** [tskit-dev/tskit#2113] [Ideas]
Hello,

As a nice resource for the community, would be nice to have the 1000 genomes / UK biobank datasets already in tskit format (trees inferred with tsinfer) somewhere. Would that be possible?

Thanks!

**Comment by giulioisac** [comment]
I guess this one is already a good starting point: https://zenodo.org/record/5495535#.YTolVdNKhTY . 
Maybe a reference to it on the documentation would help, but otherwise, thanks for the good work!

**Comment by jeromekelleher** [comment]
Hi @giulioisac :wave:

The most up-to-date inferred human tree sequences will be available when the [Unified genealogy](https://www.biorxiv.org/content/10.1101/2021.02.16.431497v2) preprint is fully published, hopefully quite soon. These will include 1000G and other datasets.

We can't share the trees we estimated for UKB chr20 I'm afraid, because of data access requirements.

**Reply by giulioisac** [reply]
I understand, thank you very much!
